contributing/using-git.txt

```diff
@@ -842,6 +842,13 @@ Pull Request.
     git push gh new_stuff
     # Now go to the new_stuff branch on github.com and open the PR
 
+GitHub's "Open a pull request" page invites you to leave a comment under
+the PR title: we use this comment to describe the PR. A good choice of
+PR title and description are both helpful to reviewers of your work. If
+there is template text already provided in the comment box for you to
```

Comment:
This last sentence is very wordy. Maybe:
"If there is template text already provided, do consider what it says but also feel free to edit it as necessary to make sense for your PR." ?